### PR TITLE
RC3 Update

### LINKED ISSUE #571: RC3 Update
- [x] Add `base58CheckEncode` and `base58CheckDecode` to class `StdLib` ([pr](https://github.com/neo-project/neo/pull/2468))
- [x] Add `getAccountState` to class `NeoToken` in devpack module ([pr](https://github.com/neo-project/neo/pull/2475))
- [x] Add `getAccountState` to class `NeoToken` in contract module ([pr](https://github.com/neo-project/neo/pull/2475))

### PR DESCRIPTION
**Pull request by mialbu**
Closes #571

### CODE REVIEW

File: devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java [hunk 1]
```diff
@@ -133,6 +133,24 @@
...
+     * of the binary data.
+     *
+     * @param input The bytes to encode.
+     * @return the encoded string.
+     */
+    public static native String base58CheckEncode(ByteString input);
+
+    /**
+     * Decodes the given Base58-encoded string. The encoded string contains the checksum of the
+     * binary data.
```
Comment (csmuller) [inline devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java:147]:
```suggestion
     * Decodes the given Base58-encoded string. Expects the input to contain the checksum of the binary data.
```

File: devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java [hunk 2]
```diff
@@ -133,6 +133,24 @@
...
+     *
+     * @param input The Base58-encoded string.
+     * @return the decoded bytes.
+     */
+    public static native String base58CheckDecode(String input);
```
Comment (csmuller) [inline devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java:152]: I think the return type should be `ByteString`.

**Review by csmuller — CHANGES_REQUESTED** [review]

File: devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java (hunk 2 above)
Comment (mialbu) [reply devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java:152]: You're right, I overlooked that.

File: devpack/src/main/java/io/neow3j/devpack/contracts/NeoToken.java [hunk 3]
```diff
@@ -95,10 +95,51 @@
...
+     *
      * @param registerPrice The new price for registering a candidate.
      */
     public static native void setRegisterPrice(int registerPrice);
 
+    /**
+     * Gets the account state.
+     *
+     * @param scriptHash The script hash of the account.
+     * @return The state of the account.
```
Comment (csmuller) [inline devpack/src/main/java/io/neow3j/devpack/contracts/NeoToken.java:107]:
```suggestion
     * @return The state of the account or null if the account was not yet used on-chain.
```

**Review by csmuller — APPROVED** [review]

**Pull request merged**